tf2_ros/src/tf2_monitor.cpp

```diff
@@ -190,6 +195,11 @@ class TFMonitor
     double lowpass = 0.01;
     unsigned int counter = 0;
 
+    if (using_specific_chain_) {
+      std::cout << "Gathering data on " << framea_ << " -> " << frameb_ << "...\n";
+    } else {
+      std::cout << "Gathering data on all frames...\n";
+    }
```

Comment:
Is it always exactly 10 seconds?  If so, I think it would be a good idea to add that to the message so that the user knows how long they have to wait.

Also, can we use `RCLCPP_INFO` here instead of `std::cout`?

Reply:
It is always 10 seconds, added to the message in b11ca5bc0df7227b7370e0f6bf3dc7ef1b775b01 . I left it as `std::cout` to stay consistent with the rest of the function.